RELEASE.md

```diff
@@ -75,24 +75,29 @@ you can skip the dependency update or only update select dependencies. In such a
 case, you have to create an issue or pull request in the GitHub project for
 later follow-up.
 
-### Prepare your release
+### 1. Prepare your release
 
-For a patch release, work in the branch of the minor release you want to patch.
+Create a new release branch.
+
+For a patch release, work in the branch of the minor release you want to patch. So for 2.16.1, create a new branch off of 2.16.0.
```

Comment:
No. 2.16.0 has the release branch `release-2.16`. All patch releases are done in the same branch.

Reply:
Yes, all releases are done from the same branch. But you still need to merge changes for the new patch release via PR into `release-2.16`.